packages/clarity-decode/src/interaction.ts

```diff
@@ -32,8 +32,7 @@ export function decode(tokens: Data.Token[]): InteractionEvent {
                 context: tokens[9] as number,
                 text: tokens[10] as string,
                 link: tokens[11] as string,
-                hash: tokens[12] as string,
-                region: tokens.length > 13 ? tokens[13] as number : null
```

Comment:
I don't think we should tkae this out as we'll lose the association of time and timeline for activiites. Let' discuss this offline.

Reply:
We reviewed this internally few weeks back. We can discuss this offline but this is part of the simplication and reducing chatter over wire